index.html

```diff
@@ -33,5 +42,141 @@ <h3>I'm a software Developer, </h3>
+          <button class="see-project btn-change"><a href="#">See project</a></button>
+        </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+    </section>
+    <section class="section-four">
+      <div class="about-me-ctn">
+        <h1>About me</h1>
+        <p>
+          Hello I&apos;m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, don&apos;t hesitate to contact me.
+        </p>
+        <button class="btn-change"><a href="#">Get my resume</a></button>
+      </div>
+    </section>
+    <section class="section-five">
+      <img class="divider" src="./final-images/Devider.png" alt="">
```

Comment:
- [ ] Well done on adding the divider to your design! However, for better semantic HTML, According to your previous reviewer I recommend using the `<hr>` tag instead of a `<div>` element to represent the divider. The `<hr> tag` is specifically designed for creating horizontal rules, which better conveys the purpose of a visual divider. Additionally, ensure that the styles for the divider align with the design specifications.
- [ ] [ See pREveviouse Review](https://github.com/prince-rutagengwa/mobile-portfolio/pull/2#discussion_r1257265958)